docs/notebooks/active_learning_for_binary_classification.pct.py

```diff
@@ -79,47 +82,56 @@ def circle(x):
 # Lets see our model landscape using only those initial data
 
 # %%
+from util.plotting import plot_bo_points, plot_function_2d
+
 model.update(initial_data)
 model.optimize(initial_data)
 
-mean, variance = model.predict(query_points)
 
-plt.figure()
-plt.contourf(*grid_query_points, np.reshape(mean, [density] * input_dim))
-plt.plot(
-    initial_data.query_points[:, 0],
-    initial_data.query_points[:, 1],
-    "ko",
-    markersize=10,
-)
-plt.title("Mean")
-plt.colorbar()
-plt.show()
+def plot_active_learning_query(function, points, num_initial_points, title):
```

Comment:
I probobaly should have pointed you to a right function, but was a bit lazy to look it up - `plot_model_predictions_plotly` does exactly what you want here, and to add acq points you can use `add_bo_points_plotly` - see for an example [here](https://github.com/secondmind-labs/trieste/blob/ce84c5e01ccbebfa82ef0490a7277445a8eb1cd2/docs/notebooks/expected_improvement.pct.py#L203)